snuba/consumers/consumer.py

```diff
@@ -344,6 +303,16 @@ def create(
             self.__max_batch_time,
         )
 
+        def process_message(
```

Comment:
There actually was a reason why process_message had to be at module level
https://github.com/getsentry/snuba/pull/1269#discussion_r478821847